guides/doc-Managing_Hosts/topics/Administering_Hosts.adoc

```diff
@@ -56,6 +57,32 @@ For the list of values, see https://access.redhat.com/documentation/en-us/red_ha
 ----
 # subscription-manager status
 ----
+
+[id="editing-the-system-purpose-of-a-host"]
+== Editing the System Purpose of Multiple Hosts
+
+You can edit the system purpose attributes of {RHEL} hosts.
+System purpose attributes define which subscriptions to attach automatically to hosts.
+For more information about system purpose, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/performing_a_standard_rhel_installation/graphical-installation_graphical-installation#configuring-system-purpose-standard_configuring-system-settings[Configuring system purpose] in the _Performing a standard RHEL installation_ guide.
+
+.Prerequisites
+
+* The hosts that you want to edit must be registered with the subscription-manager.
+
+.Procedure
+
+. In the {Project} web UI, navigate to *Hosts* > *Content Hosts* and select Red{nbsp}Hat Enterprise Linux{nbsp}8 hosts that you want to edit.
+. Click the *Select Action* button and from the drop-down menu, click *Manage System Purpose*.
```

Comment:
As per IBM style guide: Click the **Select Action** list and select **Manage System Purpose**.

Reply:
Thanks for the review, I have applied your feedback.